### ISSUE TITLE
syntax error when using ./algo

### ISSUE THREAD

**Issue by withzombies** [trailofbits/algo#78]
```bash
# ./algo

  What provider would you like to use?
    1. DigitalOcean
    2. Amazon EC2
    3. Google Compute Engine
    4. Install to existing Ubuntu server

Enter the number of your desired provider
: 4
ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/root/algo/roles/vpn/handlers/main.yml': line 1, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: restart strongswan
  ^ here


The error appears to have been in '/root/algo/roles/vpn/handlers/main.yml': line 1, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: restart strongswan
  ^ here

```

**Comment by defunctio** [comment]
resolved in 50f43dc601c32624d283f4d690fc213b64b93700
systemd is only supported in Ansible 2.2, reverted to using service for now

**Comment by dguido** [comment]
There might still be a problem [here](https://github.com/trailofbits/algo/blob/master/roles/vpn/handlers/main.yml#L2):

```
...
RUNNING HANDLER [security : restart iptables] **********************************
changed: [x.x.x.x]

RUNNING HANDLER [vpn : restart strongswan] *************************************
fatal: [x.x.x.x]: FAILED! => {"changed": false, "failed": true, "msg": "unsupported parameter for module: daemon_reload"}

RUNNING HANDLER [vpn : daemon-reload] ******************************************

RUNNING HANDLER [vpn : restart apparmor] ***************************************

NO MORE HOSTS LEFT *************************************************************

PLAY RECAP *********************************************************************
x.x.x.x              : ok=50   changed=39   unreachable=0    failed=1   
localhost                  : ok=6    changed=2    unreachable=0    failed=0   
```
